src/i18u.StartSmart.Web/app/src/api/Authentication/index.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+const host = window.location.host;
+
+const instance = axios.create({
+  baseURL: `http://${host}/api/`,
```

Comment:
Hardcoded http >:0

```suggestion
  const protocol = window.location.protocol;

  baseURL: `${protocol}//${host}/api/`,
```

`window.location.protocol` includes the colon too.